src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -920,4 +920,9 @@ public static void createSourceBQUpdateTable() throws IOException, InterruptedEx
     PluginPropertyUtils.addPluginProp(" bqTargetTable",  bqTargetTable);
     BeforeActions.scenario.write("BQ Target Table " +  bqTargetTable + " updated successfully");
   }
+
+  @Before(order = 1, value = "@GCS_CSV")
```

Comment:
Can't we use an existing csv test data?

Reply:
Existing CSV data doesn't have unique column such as (ID) as for the validation its required to store the data in a map. So key must be unique. So data has been added as per the requirement for the success step of validation.